docs/MAINTAINERS.adoc

```diff
@@ -0,0 +1,162 @@
+= Documentation for core maintainers
+
+Jenkins core is a central component of the project which serves millions of users, and ot is critical to maintain the it in a good shape.
```

Comment:
"maintain the it"?

I don't know what the sentence should be, so no suggestion from me.